e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -422,22 +422,22 @@ $iotHubName = az deployment group show -g $ResourceGroup -n $deploymentName --qu
 #################################################################################################################################################
 # Configure an AAD app and create self signed certs and get the bytes to generate more content info.
 #################################################################################################################################################
-Write-Host "`nCreating App Registration $logAnalyticsAppRegnName"
+Write-Host "`nCreating app registration $logAnalyticsAppRegnName"
 $logAnalyticsAppRegUrl = "http://$logAnalyticsAppRegnName"
-az ad sp create-for-rbac -n $logAnalyticsAppRegUrl --role "Reader" --scope $resourceGroupId --output none
-$logAnalyticsAppId = az ad app list --display-name $logAnalyticsAppRegnName --query "[?displayName=='$logAnalyticsAppRegnName'].appId" --output tsv
-Write-Host "`nApplication $logAnalyticsAppRegnName with Id $logAnalyticsAppId was created successfully."
+$logAnalyticsAppId = az ad sp create-for-rbac -n $logAnalyticsAppRegUrl --role "Reader" --scope $resourceGroupId --query "appId" --output tsv
+Write-Host "`nCreated application $logAnalyticsAppRegnName with Id $logAnalyticsAppId."
 
 #################################################################################################################################################
 # Configure an AAD app to perform IoT hub data actions.
 #################################################################################################################################################
-Write-Host "`nCreating App Registration $iotHubAadTestAppRegName"
+Write-Host "`nCreating app registration $iotHubAadTestAppRegName for IoT hub data actions"
 $iotHubAadTestAppRegUrl = "http://$iotHubAadTestAppRegName"
 $iotHubDataContributorRoleId = "4fc6c259987e4a07842ec321cc9d413f"
 $iotHubScope = "/subscriptions/$SubscriptionId/resourceGroups/$ResourceGroup/providers/Microsoft.Devices/IotHubs/$iotHubName"
-$iotHubAadTestAppPassword = az ad sp create-for-rbac -n $iotHubAadTestAppRegUrl --role $iotHubDataContributorRoleId --scope $iotHubScope --query password --output tsv
-$iotHubAadTestAppId = az ad app list --display-name $iotHubAadTestAppRegName --query "[?displayName=='$iotHubAadTestAppRegName'].appId" --output tsv
-Write-Host "`nApplication $iotHubAadTestAppRegName with Id $iotHubAadTestAppId was created successfully."
+$iotHubAadTestAppInfo = az ad sp create-for-rbac -n $iotHubAadTestAppRegUrl --role $iotHubDataContributorRoleId --scope $iotHubScope --query '{appId:appId, password:password}' | ConvertFrom-Json
```

Comment:
Instead of creating and then listing, we can create once, querying both properties we want, and then access the json object that is returned for appid and password.